geode/advanced-examples/cascading-functions/src/main/java/example/springdata/geode/functions/cascading/Address.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *
```

Comment:
Nit: License headers should look like https://github.com/spring-projects/spring-data-examples/blob/master/jpa/deferred/src/main/java/example/Application.java#L1-L15

Reply:
Updated the headers.